pkgs/development/libraries/buddy/default.nix

```diff
@@ -18,7 +18,7 @@ stdenv.mkDerivation rec {
     description = "Binary decision diagram package";
     license = "as-is";
 
-    platforms = with stdenv.lib.platforms; allBut cygwin;
+    platforms = stdenv.lib.platforms.unix; # Once had cygwin problems
```

Comment:
`unix` is actually exactly `allBut cygwin` today, but comment is still good because `cygwin` probably should be considered Unix and Windows.